src/System.Management.Automation/engine/parser/Parser.cs

```diff
@@ -2744,28 +2744,42 @@ private StatementAst SwitchStatementRule(LabelToken labelToken, Token switchToke
 
                 while (true)
                 {
-                    ExpressionAst clauseCondition = GetSingleCommandArgument(CommandArgumentContext.SwitchCondition);
-                    if (clauseCondition == null)
-                    {
-                        // ErrorRecovery: if we don't have anything that looks like a condition, we won't
-                        // find a body (because a body is just a script block, which works as a condition.)
-                        // So don't look for a body, hope we find the '}' next.
+                    Token token = PeekToken();
+                    var isDefaultClause = token.Kind == TokenKind.Default;
+                    ExpressionAst clauseCondition = null;
 
-                        isError = true;
-                        ReportIncompleteInput(After(endErrorStatement),
-                            nameof(ParserStrings.MissingSwitchConditionExpression),
-                            ParserStrings.MissingSwitchConditionExpression);
-                        // Consume a closing curly, if there is one, to avoid an extra error
-                        if (PeekToken().Kind == TokenKind.RCurly)
+                    if (isDefaultClause)
+                    {
+                        // Consume the 'default' token.
+                        SkipToken();
+                        endErrorStatement = token.Extent;
+                    }
+                    else
+                    {
+                        clauseCondition = GetSingleCommandArgument(CommandArgumentContext.SwitchCondition);
+                        if (clauseCondition == null)
                         {
-                            SkipToken();
+                            // ErrorRecovery: if we don't have anything that looks like a condition, we won't
+                            // find a body (because a body is just a script block, which works as a condition.)
+                            // So don't look for a body, hope we find the '}' next.
+
+                            isError = true;
+                            ReportIncompleteInput(After(endErrorStatement),
+                                nameof(ParserStrings.MissingSwitchConditionExpression),
+                                ParserStrings.MissingSwitchConditionExpression);
+                            // Consume a closing curly, if there is one, to avoid an extra error
```

Comment:
Please add new line.
```suggestion

                            // Consume a closing curly, if there is one, to avoid an extra error
```

Reply:
Ditto, I only indented the code block within another `if` block.   Do you still want an blank line added?

Reply:
The same.

Reply:
@msftrncs for reference, if we update a section of code (even if just by indentation) CodeFactor will consider it part of the modifications of a PR and will raise a warning about comment styles. I believe the current comment style rule is that comments should have a blank line above, but not below them.

Reply:
@vexx32, I just wanted to be sure, sometimes the preference is not to modify blocks of code that are not actually being changed in any functional way, including in reformatting, other than changing the indent.   Different repositories have different tolerances to changes such as this.

Reply:
You can ignore such minor style issues. I will fix them later per month.